platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/schedulers/internal/ConcurrentTask.java

```diff
@@ -63,6 +67,9 @@ class ConcurrentTask extends AbstractTask {
     @Override
     protected boolean exec() {
         try {
+            if (squelcher.shouldSquelch()) {
```

Comment:
I think @cody-littley was mentioning that we would want to wrap the handler with the squelcher. That way, if there is no squelching, it would call the handler directly and there would be no overhead.

Reply:
@lpetrovic05 Do you mean wrap the handler into a lambda that manages squelching? I avoided that solution, since it would add a lot of overhead creating a bunch of lambdas.

The solution I went with doesn't add any meaningful overhead if squelching is disabled for a scheduler. Schedulers with no squelching ability will have a `NoopSquelcher`, which automatically returns `false` for `shouldSquelch`.

Reply:
There is a way to do it that doesn't require a large number of lambdas. If you wrap the lambda in `BindableInputWire`, then you will only create two lambdas per input wire at start up time.

Reply:
Reworked